zulipterminal/ui_tools/boxes.py

```diff
@@ -734,6 +736,19 @@ def soup2markup(self, soup: Any, **state: Any) -> List[Any]:
                 markup.extend(self.soup2markup(element, **state))
             elif element.name == 'table':
                 markup.extend(render_table(element))
+            elif element.name == 'time':
+                # Render time in current user's local time zone.
+                timestamp = element.get('datetime')
+                if not timestamp:
+                    continue
+                utc_time = parse(timestamp)
```

Comment:
This seems like a function name lacking context, so perhaps don't import it directly?